dell-csi-helm-installer/verify-csi-isilon.sh

```diff
@@ -19,6 +19,6 @@ function verify-csi-isilon() {
   verify_alpha_snap_resources
   verify_snap_requirements
   verify_helm_3
-  verify_helm_values_version "2.5.0"
+  verify_helm_values_version "${DRIVER_VERSION}"
```

Comment:
isn't it "DEFAULT_DRIVER_VERSION" ?

Reply:
Both are same only. Driver version is taken from values.yaml